modules/persistence/src/services/metadata/ToC/utils/prefixFromEnvironment.ts

```diff
@@ -0,0 +1,9 @@
+import { ReposBranchesDocument } from '../../associated_products';
+
+export const prefixFromEnvironment = (repoBranchEntry: ReposBranchesDocument) => {
+  const env = process.env.SNOOTY_ENV ?? 'dotcomprd';
```

Comment:
minor question. why is dotcomprd default prefix setting?

Reply:
Safest failure option in the event that a production job run encounters a nullish `SNOOTY_ENV` value.